src/corelibs/U2View/src/ov_sequence/DetViewSequenceEditor.cpp

```diff
@@ -360,6 +360,10 @@ void DetViewSequenceEditor::sl_paste(Task *task) {
 
     U2OpStatusImpl os;
     const QList<DNASequence> &sequences = PasteUtils::getSequences(docs, os);
+    if (sequences.isEmpty()) {
+        coreLog.error(tr("No sequences detected in pasted content."));
```

Comment:
I would recommend using a simpler "No sequences found" or "No sequences detected in the pasted content."